src/views/Dashboard/TooltipCard.js

```diff
@@ -7,6 +7,7 @@ export default function TooltipCard({
   arbitrum,
   showDollar = true,
   decimalsForConversion = USD_DECIMALS,
+  symbol,
```

Comment:
i missed PR in which TooltipCard was added, i think views should not contain components that are used inside other views or other components. 

e.g. Exchange view requires it
`import { TooltipCardRow } from "../../views/Dashboard/TooltipCard";`

Can we move TooltipCard to components/Tooltip dir?

Reply:
Sure I will do that. It just stated as to be used only on the DashboardV2 later we wanted to use it everywhere, to make all the tooltip look similar.

Done!